From 963a609921c1f8394cfa1a8d073a5a02b549c712 Mon Sep 17 00:00:00 2001
From: Ulrich Hecht <uli@basicengine.org>
Date: Wed, 4 Oct 2023 17:44:09 +0200
Subject: [PATCH 039/184] 
 linux-mainline-and-mali-allwinner-h6-kernel/misc.616/patches/add-own-dtb-files.patch

---
 arch/arm64/boot/dts/allwinner/Makefile | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/Makefile b/arch/arm64/boot/dts/allwinner/Makefile
index 7b8853a5a..76fcb7a15 100644
--- a/arch/arm64/boot/dts/allwinner/Makefile
+++ b/arch/arm64/boot/dts/allwinner/Makefile
@@ -42,3 +42,10 @@ dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-orangepi-zero2.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-x96-mate.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-tanix-tx6s.dtb
 dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-t95.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h313-x96q.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-tx6s.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-lemfo.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-t95max.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h313-x96-q-lpddr3.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h616-tanix-tx6s-axp313.dtb
+dtb-$(CONFIG_ARCH_SUNXI) += sun50i-h618-vontar-h618.dtb
-- 
2.39.2

